PusherChatkit.xcodeproj/project.pbxproj

```diff
@@ -617,15 +613,15 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
```

Comment:
Same as comment above.

Reply:
This setting is for the test target. The framework target was specifying 10.0.